addons/bestja_detailed_reports/menu.xml

```diff
@@ -0,0 +1,10 @@
+<openerp>
+    <data>
+        <menuitem parent="bestja_base.menu_bestja" id="menu_detailed_reports" name="Raporty szczegółowe" sequence="10" groups="bestja_project.managers"/>
+        <menuitem parent="menu_detailed_reports" id="menu_detailed_reports_profile" action="action_bestja_detailed_reports" name="Moje raporty szczegółowe" sequence="1" groups="bestja_project.managers"/>
```

Comment:
Are all those menu items supposed to simultaneously be the first (`sequence="1"`)?